ingestr/src/google_analytics/helpers.py

```diff
@@ -170,3 +239,54 @@ def _resolve_dimension_value(dimension_name: str, dimension_value: str) -> Any:
         return pendulum.from_format(dimension_value, "YYYYMMDDHHmm", tz="UTC")
     else:
         return dimension_value
+    
+def convert_minutes_ranges_to_int_list(minutes_ranges: str) -> List[int]:
```

Comment:
I wouldn't return this as a list of integers, instead I would make this return already a list of minute-range objects. that way if google supports more minute ranges tomorrow this would work automatically.